CouchDocument/AuthCodeManager.php

```diff
@@ -81,15 +81,6 @@ public function deleteAuthCode(AuthCodeInterface $authCode)
      */
     public function deleteExpired()
     {
-        $result = $this
-            ->repository
-            ->createQueryBuilder()
-            ->remove()
-            ->field('expiresAt')->lt(time())
-            ->getQuery(['safe' => true])
-            ->execute()
-	;
-
-        return $result['n'];
+        return null;
```

Comment:
Why doesn't the existing code work?

Reply:
CouchDB work with JS view to make request, I don't have function to request and check date, I have to find a way to make this view and import this view in CouchDB.

Reply:
Yes, but ODM abstracts all of it away. My questions was: why can't existing code (or something very similar to it) do the same task.